static/css/style.css

```diff
@@ -1,165 +1,68 @@
 /*  import google fonts */
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Ubuntu:wght@400;500;700&display=swap');
-
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@200&display=swap');
+/*general styling*/
 *{
     margin: 0;
     padding: 0;
     box-sizing: border-box;
     text-decoration: none;
+    font-family: 'Oswald', sans-serif;
+    font-weight: bold;
 }
-*, *:before, *:after {
-    box-sizing: inherit;
-}
-
 html{
     scroll-behavior: smooth;
-    box-sizing: border-box;
-}
-*, *:before, *:after {
-    box-sizing: inherit;
-  }
-/* home section styling */
-.home{
-    color: black;
-    background-size: cover;
-    background-color: gainsboro;
-    background-attachment: fixed;
-    font-family: 'Ubuntu', sans-serif;
-}
-.home h1{
-    margin:40px;
-    padding:40px;
-    text-align: center;
-    font-size: 100px;
-}
-
-/*about us page*/
-
-.img-class{
-    width: 100%;
-}
-
-.img-class img{
-    width:100%;
-    border-radius: 50%;
-    box-shadow: 0 0 12px white;
 }
-
-.ab-container{
-    height: 70vh;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-}
-
-.about-us-h{
-    font-size: 50px;
-    color:#888;
-    padding:20px;
-    margin:20px;
-    justify-content: center;
-    text-align: center;
+body, html {
+    height: 100%;
 }
 
-.about-us-p{
-    font-size: 20px;
-    color:#888;
-    padding:40px;
-    margin:20px;
-    justify-content: center;
-    text-align: center;
+/* Nav Bar */
+.navbar {
+    background: #111;
+    font-family: 'Oswald', sans-serif;
+    font-size:24px;
+    font-weight:700 ;
+    letter-spacing: .1rem;
+}
+.nav-item {
+    padding:.7rem; /*8px*/
+}
+.logo{
+    width:150px;
+    height:100px;
+}
+.nav-link:hover{
+    color:red !important;
+}
+/* The hero image */
+.hero-image {
+    background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),url("../images/community.jpg");
```

Comment:
it would be better to use `<img>` with a proper alt text

Reply:
> I need you to please add a boolean variable called passed to the model and a static method that returns a chore by id, I need those for the Vote model

done.
added the passed file and the static method.

Reply:
yeah i had issues with the image making it fit..so yeh had to keep it the same..